src/Estonianpin/Utils/Utils.php

```diff
@@ -239,4 +239,37 @@ public function generateRandomDateArray(): array {
         ];
     }
 
+    /**
+     * @param \DateTime $start
+     * @param \DateTime $end
+     * @return Iterable
+     */
```

Comment:
Please provide full documentation in this doc block. Refer to other doc blocks in this project.